ldes-server-infra-mongo/mongo-retention-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/retention/MemberPropertiesRepositoryImpl.java

```diff
@@ -66,12 +60,14 @@ public Optional<MemberProperties> retrieve(String id) {
 	}
 
 	@Override
-	public synchronized void addViewReference(String id, String viewName) {
-		Query query = new Query();
-		query.addCriteria(Criteria.where(ID).is(id));
-		Update update = new Update();
-		update.addToSet(VIEWS, viewName);
-		mongoTemplate.upsert(query, update, MemberPropertiesEntity.class);
+	public void addViewToAll(ViewName viewName) {
+		final var query = new Query();
```

Comment:
Shouldn't the view not only be added to the members that are in the collection whereto the view is added?

Reply:
This was indeed a bug..